### DISCUSSION TITLE
抖音视频弹幕浅色背景显示不清晰需要如何设置？

### DISCUSSION THREAD

**Discussion by Leslie90s** [F9y4ng/GreasyFork-Scripts#346] [问答专区(Question / Answer)] [answered]
抖音视频弹幕浅色背景显示不清晰需要如何设置？

**Comment by F9y4ng** [comment, marked as answer]
“抖音视频弹幕”，我猜测你所指的应该是直播弹幕吧？

关闭**字体描边**以及**字体阴影**可以解决你遇到的问题。（设置为0）

**Reply by Leslie90s** [reply]
抖音网页版短视频和直播都有弹幕，以为要像B站一样，需要在**需要渲染的网页元素**添加代码解决，实测只关闭**字体阴影**即可。

**Reply by F9y4ng** [reply]
并不同，抖音的弹幕字体相关样式非常混乱，无法通过样式筛选来实现。

**Reply by Leslie90s** [reply]
好吧，现在关闭字体阴影已经可以了，感谢大佬！

**Reply by F9y4ng** [reply]
如果问题解决，记得点击`Mark as answer`.
